### DISCUSSION TITLE
Where does `messaged config` come from?

### DISCUSSION THREAD

**Discussion by TilmannBach** [renovatebot/renovate#14333] [Request Help] [answered]
Where does `messaged config` come from?

I have some CLI parameters, and a `renovate.json` in my **self hosted** renovate bot repository on **self hosted gitlab platform**.

All my desired configuration gots printed out at the beginning.

But for my repositories my global settings are only partially respected:
```log
DEBUG: Found renovate.json config file (repository=faro/backend/faro-core)
DEBUG: Repository config (repository=faro/backend/faro-core)
       "fileName": "renovate.json",
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: migrateAndValidate() (repository=faro/backend/faro-core)
DEBUG: No config migration necessary (repository=faro/backend/faro-core)
DEBUG: massaged config (repository=faro/backend/faro-core)
       "config": {
         "$schema": "https://docs.renovatebot.com/renovate-schema.json",
         "extends": ["config:base", ":disableRateLimiting"]
       }
```

so it respects the global setting `:disableRateLimiting` but not my other settings such as `ignorePaths`.

Can someone help me out here?

**Comment by rarkins** [comment, marked as answer]
It's mAssaged, not mEssaged. As with all log message, you can search the source code for that phrase whenever you're curious. You should know that it's printing out the repo config (what you commit to your repo) at that point and not the global (self-hosted) config.

**Reply by TilmannBach** [reply]
hi @rarkins thank you for your clarification. i searched the code and could find the location but i didnt understand that it only prints out the repo config and not the final merged config (global+repo).

**Reply by viceice** [reply]
This is by design. If you like the full config, checkout docs

https://docs.renovatebot.com/self-hosted-configuration/#printconfig

**Reply by rarkins** [reply]
You can set `printConfig=true` if you want a dump of the full, resolved config. Warning: it can be quite long
